Dev build is crashing

Hi

I am using the Fdroid version of the dev build and I just upgraded it to `20200110` Prior to the upgrade I have experienced no crashes, I cant tell what the version was propbably it was the one before.

I am using it on LineageOS 14.1

I open the app, and it crashes right away, I was able to capture the log below. PLease let me know if you need more log stuff



```

1-20 11:25:25.321  9730  9730 I SQL     : Entering in the #53 add rich workspace to file table
01-20 11:25:25.322  9730  9730 E SQLiteLog: (1) duplicate column name: rich_workspace
01-20 11:25:25.323  9730  9730 D AndroidRuntime: Shutting down VM
01-20 11:25:25.328  3374 10025 I ActivityManager: START u0 {flg=0x10000000 cmp=com.nextcloud.android.beta/com.nextcloud.client.errorhandling.ShowErrorActivity (has extras)} from uid 10470 on display 0
01-20 11:25:25.340  9730  9730 E AndroidRuntime: FATAL EXCEPTION: main
01-20 11:25:25.340  9730  9730 E AndroidRuntime: Process: com.nextcloud.android.beta, PID: 9730
01-20 11:25:25.340  9730  9730 E AndroidRuntime: java.lang.RuntimeException: Unable to create application com.owncloud.android.MainApp: android.database.sqlite.SQLiteException: duplicate column name: rich_workspace (code 1): , while compiling: ALTER TABLE filelist ADD COLUMN rich_workspace TEXT
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5451)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at de.robv.android.xposed.XposedBridge.invokeOriginalMethodNative(Native Method)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at de.robv.android.xposed.XposedBridge.handleHookedMethod(XposedBridge.java:360)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.app.ActivityThread.handleBindApplication(<Xposed>)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.app.ActivityThread.-wrap2(ActivityThread.java)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1564)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:102)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:154)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6186)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:107)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: Caused by: android.database.sqlite.SQLiteException: duplicate column name: rich_workspace (code 1): , while compiling: ALTER TABLE filelist ADD COLUMN rich_workspace TEXT
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:889)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:500)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:588)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:58)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.database.sqlite.SQLiteStatement.<init>(SQLiteStatement.java:31)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.database.sqlite.SQLiteDatabase.executeSql(SQLiteDatabase.java:1677)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.database.sqlite.SQLiteDatabase.execSQL(SQLiteDatabase.java:1608)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at com.owncloud.android.providers.FileContentProvider$DataBaseHelper.onUpgrade(FileContentProvider.java:2092)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.database.sqlite.SQLiteOpenHelper.getDatabaseLocked(SQLiteOpenHelper.java:256)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.database.sqlite.SQLiteOpenHelper.getReadableDatabase(SQLiteOpenHelper.java:187)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at com.owncloud.android.providers.FileContentProvider.query(FileContentProvider.java:471)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.content.ContentProvider.query(ContentProvider.java:1020)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.content.ContentProvider$Transport.query(ContentProvider.java:239)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.content.ContentResolver.query(ContentResolver.java:534)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.content.ContentResolver.query(ContentResolver.java:475)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at com.owncloud.android.datamodel.UploadsStorageManager.getUploads(UploadsStorageManager.java:289)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at com.owncloud.android.datamodel.UploadsStorageManager.getFailedUploads(UploadsStorageManager.java:368)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at com.owncloud.android.utils.FilesSyncHelper.restartJobsIfNeeded(FilesSyncHelper.java:194)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at com.owncloud.android.MainApp.initSyncOperations(MainApp.java:492)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at com.owncloud.android.MainApp.onCreate(MainApp.java:309)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1025)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5448)
01-20 11:25:25.340  9730  9730 E AndroidRuntime: 	... 12 more
01-20 11:25:25.348  3374  9762 W DropBoxManagerService: Dropping: data_app_crash (3410 > 0 bytes)
01-20 11:25:25.444  3374 13402 I OpenGLRenderer: Initialized EGL, version 1.4
01-20 11:25:25.444  3374 13402 D OpenGLRenderer: Swap behavior 1
01-20 11:25:25.451  3374 13402 D mali_winsys: EGLint new_window_surface(egl_winsys_display*, void*, EGLSurface, EGLConfig, egl_winsys_surface**, egl_color_buffer_format*, EGLBoolean) returns 0x3000,  [1590x733]-format:1
01-20 11:25:25.838  3374  3387 W ActivityManager: Activity pause timeout for ActivityRecord{ad144d8 u0 com.nextcloud.android.beta/com.owncloud.android.ui.activity.FileDisplayActivity t56366}
01-20 11:25:25.881  3374  3387 I ActivityManager: Start proc 9763:com.nextcloud.android.beta:crash/u0a470 for activity com.nextcloud.android.beta/com.nextcloud.client.errorhandling.ShowErrorActivity



```

Same here:
OnePlus 3, Android 9, Oxygen OS-Version 9.0.6, Build-Nummer ONEPLUS A3003_16_1911004
Dont know what logfiles are helpfull, my phone dropped me a bugreport but that one is more than 30 MB.

Here some lines looking very similar to the ones from @gerroon 

`01-20 18:12:55.403  1000  1253  2409 I ActivityManager: START u0 {flg=0x10000000 cmp=com.nextcloud.android.beta/com.nextcloud.client.errorhandling.ShowErrorActivity (has extras)} from uid 10116 pid 9781
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: FATAL EXCEPTION: main
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: Process: com.nextcloud.android.beta, PID: 9781
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: java.lang.RuntimeException: Unable to create application com.owncloud.android.MainApp: android.database.sqlite.SQLiteException: duplicate column name: rich_workspace (code 1 SQLITE_ERROR): , while compiling: ALTER TABLE filelist ADD COLUMN rich_workspace TEXT
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6044)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.app.ActivityThread.access$1300(ActivityThread.java:207)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1748)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:193)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6863)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: Caused by: android.database.sqlite.SQLiteException: duplicate column name: rich_workspace (code 1 SQLITE_ERROR): , while compiling: ALTER TABLE filelist ADD COLUMN rich_workspace TEXT
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:903)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:514)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:588)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:58)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.database.sqlite.SQLiteStatement.<init>(SQLiteStatement.java:31)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.database.sqlite.SQLiteDatabase.executeSql(SQLiteDatabase.java:1819)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.database.sqlite.SQLiteDatabase.execSQL(SQLiteDatabase.java:1747)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at com.owncloud.android.providers.FileContentProvider$DataBaseHelper.onUpgrade(FileContentProvider.java:2092)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.database.sqlite.SQLiteOpenHelper.getDatabaseLocked(SQLiteOpenHelper.java:398)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.database.sqlite.SQLiteOpenHelper.getReadableDatabase(SQLiteOpenHelper.java:322)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at com.owncloud.android.providers.FileContentProvider.query(FileContentProvider.java:471)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.content.ContentProvider.query(ContentProvider.java:1138)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.content.ContentProvider.query(ContentProvider.java:1230)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.content.ContentProvider$Transport.query(ContentProvider.java:251)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.content.ContentResolver.query(ContentResolver.java:847)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.content.ContentResolver.query(ContentResolver.java:762)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.content.ContentResolver.query(ContentResolver.java:720)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at com.owncloud.android.datamodel.UploadsStorageManager.getUploads(UploadsStorageManager.java:289)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at com.owncloud.android.datamodel.UploadsStorageManager.getFailedUploads(UploadsStorageManager.java:368)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at com.owncloud.android.utils.FilesSyncHelper.restartJobsIfNeeded(FilesSyncHelper.java:194)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at com.owncloud.android.MainApp.initSyncOperations(MainApp.java:492)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at com.owncloud.android.MainApp.onCreate(MainApp.java:309)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1165)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6039)
01-20 18:12:55.433 10116  9781  9781 E AndroidRuntime: 	... 8 more
`

Same for me. I just updated, and it refuses to start.  I have been running the f-droid dev build for a very long time without any problems, this is the first time I (remember I) have had any problems with it. An older nexus 7 (flo), 3.4.0-lineageos-g90f7354.

```shell
01-20 22:29:07.319  1737  1751 E SQLiteLog: (1) duplicate column name: rich_workspace
01-20 22:29:07.321  1737  1751 E DatabaseUtils: Writing exception to parcel
01-20 22:29:07.321  1737  1751 E DatabaseUtils: android.database.sqlite.SQLiteException: duplicate column name: rich_workspace (code 1): , while compiling: ALTER TABLE filelist ADD COLUMN rich_workspace TEXT
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:889)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:500)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:588)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:58)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at android.database.sqlite.SQLiteStatement.<init>(SQLiteStatement.java:31)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at android.database.sqlite.SQLiteDatabase.executeSql(SQLiteDatabase.java:1678)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at android.database.sqlite.SQLiteDatabase.execSQL(SQLiteDatabase.java:1608)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at com.owncloud.android.providers.FileContentProvider$DataBaseHelper.onUpgrade(FileContentProvider.java:2092)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at android.database.sqlite.SQLiteOpenHelper.getDatabaseLocked(SQLiteOpenHelper.java:256)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at android.database.sqlite.SQLiteOpenHelper.getReadableDatabase(SQLiteOpenHelper.java:187)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at com.owncloud.android.providers.FileContentProvider.query(FileContentProvider.java:471)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at android.content.ContentProvider.query(ContentProvider.java:1022)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at android.content.ContentProvider$Transport.query(ContentProvider.java:240)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at android.content.ContentProviderNative.onTransact(ContentProviderNative.java:112)
01-20 22:29:07.321  1737  1751 E DatabaseUtils:         at android.os.Binder.execTransact(Binder.java:565)
01-20 22:29:07.323  1780  1780 D AndroidRuntime: Shutting down VM
01-20 22:29:07.324  1780  1780 E AndroidRuntime: FATAL EXCEPTION: main
01-20 22:29:07.324  1780  1780 E AndroidRuntime: Process: com.nextcloud.android.beta:crash, PID: 1780
01-20 22:29:07.324  1780  1780 E AndroidRuntime: java.lang.RuntimeException: Unable to create application com.owncloud.android.MainApp: android.database.sqlite.SQLiteException: duplicate column name: rich_workspace (code 1): , while compiling: ALTER TABLE filelist ADD COLUMN rich_workspace TEXT
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5451)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at android.app.ActivityThread.-wrap2(ActivityThread.java)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1564)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:102)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:154)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6186)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
01-20 22:29:07.324  1780  1780 E AndroidRuntime: Caused by: android.database.sqlite.SQLiteException: duplicate column name: rich_workspace (code 1): , while compiling: ALTER TABLE filelist ADD COLUMN rich_workspace TEXT
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:179)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:135)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at android.content.ContentProviderProxy.query(ContentProviderNative.java:421)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at android.content.ContentResolver.query(ContentResolver.java:534)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at android.content.ContentResolver.query(ContentResolver.java:475)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at com.owncloud.android.datamodel.UploadsStorageManager.getUploads(UploadsStorageManager.java:289)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at com.owncloud.android.datamodel.UploadsStorageManager.getFailedUploads(UploadsStorageManager.java:368)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at com.owncloud.android.utils.FilesSyncHelper.restartJobsIfNeeded(FilesSyncHelper.java:194)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at com.owncloud.android.MainApp.initSyncOperations(MainApp.java:492)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at com.owncloud.android.MainApp.onCreate(MainApp.java:309)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1025)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5448)
01-20 22:29:07.324  1780  1780 E AndroidRuntime:        ... 8 more
01-20 22:29:07.326   552  1335 W ActivityManager:   Force finishing activity com.nextcloud.android.beta/com.nextcloud.client.errorhandling.ShowErrorActivity
01-20 22:29:07.328   552   578 W WindowManager: Failed looking up window
01-20 22:29:07.328   552   578 W WindowManager: java.lang.IllegalArgumentException: Requested window android.view.ViewRootImpl$W@31a9272 does not exist
01-20 22:29:07.328   552   578 W WindowManager:         at com.android.server.wm.WindowManagerService.windowForClientLocked(WindowManagerService.java:9475)
01-20 22:29:07.328   552   578 W WindowManager:         at com.android.server.wm.WindowManagerService.windowForClientLocked(WindowManagerService.java:9466)
01-20 22:29:07.328   552   578 W WindowManager:         at com.android.server.wm.WindowManagerService.removeWindow(WindowManagerService.java:2402)
01-20 22:29:07.328   552   578 W WindowManager:         at com.android.server.wm.Session.remove(Session.java:202)
01-20 22:29:07.328   552   578 W WindowManager:         at android.view.ViewRootImpl.dispatchDetachedFromWindow(ViewRootImpl.java:3290)
01-20 22:29:07.328   552   578 W WindowManager:         at android.view.ViewRootImpl.doDie(ViewRootImpl.java:5923)
01-20 22:29:07.328   552   578 W WindowManager:         at android.view.ViewRootImpl$ViewRootHandler.handleMessage(ViewRootImpl.java:3629)
01-20 22:29:07.328   552   578 W WindowManager:         at android.os.Handler.dispatchMessage(Handler.java:102)
01-20 22:29:07.328   552   578 W WindowManager:         at android.os.Looper.loop(Looper.java:154)
01-20 22:29:07.328   552   578 W WindowManager:         at android.os.HandlerThread.run(HandlerThread.java:61)
01-20 22:29:07.328   552   578 W WindowManager:         at com.android.server.ServiceThread.run(ServiceThread.java:46)
01-20 22:29:07.331   552  1335 W ActivityManager:   Force finishing activity com.nextcloud.android.beta/com.nextcloud.client.errorhandling.ShowErrorActivity
01-20 22:29:07.331   552  1335 W ActivityManager: Duplicate finish request for ActivityRecord{eb44b6c u0 com.nextcloud.android.beta/com.nextcloud.client.errorhandling.ShowErrorActivity t2803 f}
01-20 22:29:07.335   552  1831 W DropBoxManagerService: Dropping: data_app_crash (2173 > 0 bytes)
01-20 22:29:07.832   552   571 W ActivityManager: Activity pause timeout for ActivityRecord{eb44b6c u0 com.nextcloud.android.beta/com.nextcloud.client.errorhandling.ShowErrorActivity t2803 f}
01-20 22:29:07.834   552   571 I WindowManager: Failed to capture screenshot of Token{3473535 ActivityRecord{eb44b6c u0 com.nextcloud.android.beta/com.nextcloud.client.errorhandling.ShowErrorActivity t2803 f}} appWin=Window{5d6e3e9 u0 Starting com.nextcloud.android.beta} drawState=4

```

--- a/src/main/java/com/owncloud/android/providers/FileContentProvider.java
+++ b/src/main/java/com/owncloud/android/providers/FileContentProvider.java
@@ -2089,9 +2089,11 @@ public class FileContentProvider extends ContentProvider {
                 Log_OC.i(SQL, "Entering in the #53 add rich workspace to file table");
                 db.beginTransaction();
                 try {
-                    db.execSQL(ALTER_TABLE + ProviderTableMeta.FILE_TABLE_NAME +
-                                   ADD_COLUMN + ProviderTableMeta.FILE_RICH_WORKSPACE + " TEXT ");
-
+                    if (!checkIfColumnExists(db, ProviderTableMeta.FILE_TABLE_NAME,
+                                             ProviderTableMeta.FILE_RICH_WORKSPACE)) {
+                        db.execSQL(ALTER_TABLE + ProviderTableMeta.FILE_TABLE_NAME +
+                                       ADD_COLUMN + ProviderTableMeta.FILE_RICH_WORKSPACE + " TEXT ");
+                    }
                     upgraded = true;
                     db.setTransactionSuccessful();
                 } finally {

